webapp/security/models.py

```diff
@@ -75,7 +75,7 @@
     Column(
         "package_release_status_id",
         Integer,
-        ForeignKey("package_release_status.id"),
+        ForeignKey("package_release_status.id", ondelete="CASCADE"),
```

Comment:
Does this mean deleting a CVE will delete a package? Or that deleting a package will delete a CVE? Are we sure we want either?

What happens if a package is referenced in multiple CVEs and then a CVE is deleted?

Reply:
So, when we perform a `Delete` operation on the db, this should delete all `package release status` entries associated with that specific package, because if they are orphaned, this data is useless. Unlike e.g. `releases` data, which are useful on their own and used by others. As far as I know this one is applied on a package release status, not a CVE, therefore, if we delete a CVE nothing will happen, if we delete a package, all these release status will be deleted too. But I can confirm it doing an db operation see if that happens correctly.

Reply:
cool. Yeah just check it works as you expect.